workflow-templates/pr-feedback.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Ask for feedback on PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
Random to avoid everything firing at the same moment

Reply:
no I don't think it makes sense to leave people out who already gave their feedback, let's include everyone, especially frequent contributors should have all possibilities to reach out

Reply:
@skjnldsv That's what I also said for the start date, otherwise people get many notifications for all their PRs since the start date at once which can be very annoying.